pkg/controller/shared/shared_informer.go

```diff
@@ -35,6 +36,8 @@ type InformerFactory interface {
 	ImageStreams() ImageStreamInformer
 	SecurityContextConstraints() SecurityContextConstraintsInformer
 	ClusterResourceQuotas() ClusterResourceQuotaInformer
+
+	KubernetesInformers() informers.SharedInformerFactory
```

Comment:
This is superseding all the upstream informers that are already exposed by our factory.